recipes-iec61850/iec61850-open-server/iec61850-open-server.bb

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2021, Alliander (http://www.alliander.com)
+# SPDX-License-Identifier: Apache-2.0
+
+LICENSE = "GPL-3.0"
+LIC_FILES_CHKSUM = "file://COPYING;md5=d32239bcb673463ab874e80d47fae504"
+
+SRC_URI = "git://github.com/robidev/iec61850_open_server.git;protocol=https;branch=master"
+
+#PV = "1.0+git${SRCPV}" # TODO check if this should be included
+SRCREV = "${AUTOREV}"
+DEPENDS = "libiec61850"
+RDEPENDS_${PN} = "libiec61850" 
+
+S = "${WORKDIR}/git"
+
+inherit cmake
+
+# NOTE: this project uses a dual license scheme, as libiec61850 is covered by GPL 3.0
```

Comment:
Is it your own comment or autogenerated by devtool ?

Reply:
This is my own comment. I included it as to make clear what is licensed under what. I would like to adhere to Apache 2, but as far as I understood, I'm not allowed to, for the parts that include libiec61850, as its licensed under GPL